end-to-end-tests/cypress/e2e/OSCAL/OSCAL_Textfield_Tests.cy.js

```diff
@@ -41,24 +38,34 @@ describe("Cancel on ESC Keypress", () => {
   before(() => {
     cy.getTestSspJson().then((result) => (origSspJson = result));
   });
-  after(() => {
+  afterEach(() => {
     cy.setTestSspJson(origSspJson);
   });
 
   it("ESC key cancels any changes", () => {
-    cy.navToTestSspRestMode(SSP_TITLE_ORIG);
+    cy.navToTestSspRestMode(SSP_ALTERNATE);
+        cy.get(`[aria-label="edit-system-security-plan-metadata-title"]`).click();
+
     cy.get(`[data-testid="textField-system-security-plan-metadata-title"]`)
       .click()
       .clear()
-      .type("My new title {esc}")
-      .should("have.value", SSP_TITLE_ORIG);
+      .type("My new title {esc}");
+      cy.get(`[aria-label="edit-system-security-plan-metadata-title"]`).click();
+      cy.get(`[data-testid="textField-system-security-plan-metadata-title"]`)
+      .click()
+      .should("have.value", SSP_ALTERNATE);
   });
   it("Enter key changes textfield value", () => {
-    cy.navToTestSspRestMode(SSP_TITLE_ORIG);
+    cy.navToTestSspRestMode(SSP_ALTERNATE);
+    cy.get(`[aria-label="edit-system-security-plan-metadata-title"]`).click();
+
     cy.get(`[data-testid="textField-system-security-plan-metadata-title"]`)
       .click()
       .clear()
-      .type("{esc}")
-      .should("have.value", SSP_TITLE_ORIG);
+      .type("{esc}");
+      cy.get(`[aria-label="edit-system-security-plan-metadata-title"]`).click();
+      cy.get(`[data-testid="textField-system-security-plan-metadata-title"]`)
+      .click()
+      .should("have.value", SSP_ALTERNATE);
```

Comment:
Formatting is very weird here. The `cy.get` lines likely need to be un-indented? Same in a few other places.